src/video/windows/SDL_windowsevents.c

```diff
@@ -718,11 +718,12 @@ WIN_WindowProc(HWND hwnd, UINT msg, WPARAM wParam, LPARAM lParam)
                     (GetMouseMessageSource() == SDL_MOUSE_EVENT_SOURCE_TOUCH || (GetMessageExtraInfo() & 0x82) == 0x82)) {
                     break;
                 }
+                const SDL_MouseID mouseID = (SDL_MouseID)(uintptr_t)inp.header.hDevice;
```

Comment:
This will hit `-Werror=declaration-after-statement`

Reply:
Indeed, fixed.